### PR TITLE
[gcp setup] tell part time students to skip gcp setup

### PR DESCRIPTION
**Pull request by gmanchon**
Following https://github.com/lewagon/data-setup/issues/82

### CODE REVIEW

**Comment by gmanchon** [comment]
@brunolajoie should we add a remainder in the challenges for the part-time students to complete the setup ?

**Comment by gmanchon** [comment]
@brunolajoie we discussed we @krokrob and thought we might couple this with a message to the teacher team from data-meta. I'll look into this

File: _partials/gcp_setup.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 [GCP](https://cloud.google.com/) is a cloud solution that you are going to use in order to deploy your Machine Learning-based products to production.
 
+🚨 If you are a student of the **Part-Time Bootcamp**, skip this section for now! **GCP** offers $300 worth of free credits for a duration of 3 months. You do not want to activate your GCP account too soon 🙅‍♂️
```
Comment (krokrob) [inline _partials/gcp_setup.md:5]:
```suggestion
🚨 If you are a student of the **Part-Time Bootcamp**, SKIP THIS SECTION FOR NOW! **GCP** offers $300 worth of free credits for a duration of 3 months. You do not want to activate your GCP account too soon 🙅‍♂️
```

**Review by krokrob — APPROVED** [review]
👍

**Pull request merged**